style.css

```diff
@@ -429,3 +514,260 @@ input:focus:invalid {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+  }
+
+  .notif-bar a {
+    color: #fff;
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  /* introduction page */
+
+  .introduction {
+    background-image: url(images/art\ 1.png);
+    padding: 0 0 300px 36px;
+    display: flex;
+    background-size: 40%;
+    background-position: right;
+  }
+
+  .social-Dskt {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: baseline;
+    gap: 20px;
+    width: fit-content;
+    padding-right: 100px;
+    padding-top: 351px;
+  }
+
+  .intro-Dskt {
+    width: 39.3%;
+    height: fit-content;
+    padding-top: 264px;
+  }
+
+  .introduction h1 {
+    width: 100%;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+  }
+
+  .introduction h3 {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+  }
+
+  .introduction p {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  /* My recent works page */
+
+  .Second-Page {
+    padding: 130px 141px;
+  }
+
+  .Works h2 {
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #091e42;
+    width: fit-content;
+    padding-right: 24px;
+    margin-bottom: -35px;
+  }
+
+  .Works hr {
+    margin-left: 343px;
+  }
+
+  .middle-part {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 24px;
+    width: 100%;
+  }
+
+  .multi-post {
+    width: 40.87%;
+  }
+
+  .Works img {
+    width: 62.4%;
+  }
+
+  .Works h3 {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+  }
+
+  .Works p {
+    font-size: 20px;
+    line-height: 28px;
+    overflow: hidden;
+  }
+
+  .Works ul {
+    width: 297px;
+    height: 40px;
+  }
+
+  .dsktp-cards img {
+    display: flex;
+    flex-direction: row;
+    width: 31.94%;
+    padding-top: 24px;
+  }
+
+  .group1 {
+    display: flex;
+    flex-direction: row;
+    gap: 24px;
+    padding-bottom: 37px;
+  }
+
+  .group2 {
+    display: flex;
+    flex-direction: row;
+    gap: 24px;
+  }
+
+  /* About me page */
+
+  .Third-page {
+    background-image: url(images/illustration-about\ me@2x.png);
+    background-size: 35%;
+    background-color: #262626;
+    background-position: right;
+    padding: 130px 10% 139px 10%;
+  }
+
+  .About-Me {
+    width: 48.09%;
+  }
+
+  .About-Me h2 {
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+  }
+
+  .About-Me p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
```

Comment:
- [ ] In the about section, kindly adjust the background image present to be expanded and rightly aligned as it is on the Figma design
- Presently, you need to center it. This has more appealing and shows a professional view on user experience consideration 👍 

## View:
![image](https://user-images.githubusercontent.com/50990925/228419267-7e5153b5-9247-428e-b607-3fc7c146f93a.png)

## Figma
![image](https://user-images.githubusercontent.com/50990925/228419301-60973fb3-fd2c-4f49-8a01-d3c25c142312.png)